apps/site/src/lib/admin/useApplicant.ts

```diff
@@ -40,8 +40,13 @@ export enum ReviewStatus {
 	released = "RELEASED",
 }
 
-export const Status = { ...ReviewStatus, ...Decision };
-export type Status = ReviewStatus | Decision;
+export enum WaiverStatus {
+	signed = "WAIVER_SIGNED",
+	confirmed = "CONFIRMED",
```

Comment:
I'm assuming we want postAcceptedStatus = "CONFIRMED" and not the other way around, correct?

Reply:
Oh sorry, I should have been clearer. I meant change `WaiverStatus` to `PostAcceptedStatus`, since both of these statuses can only be for people that have been accepted.

Reply:
ah got it